DataFormats/TrackingRecHitSoA/src/alpaka/classes_cuda.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DataFormats_TrackingRecHitSoA_alpaka_alasses_cuda_h
+#define DataFormats_TrackingRecHitSoA_alpaka_alasses_cuda_h
+
+#include "DataFormats/Common/interface/DeviceProduct.h"
+#include "DataFormats/Common/interface/Wrapper.h"
+#include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsSoA.h"
+#include "DataFormats/TrackingRecHitSoA/interface/alpaka/TrackingRecHitsSoACollection.h"
+#include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsDevice.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+using namespace pixelTopology;
```

Comment:
the `classes_rocm.h` doesn't have this line, maybe it is not needed?

Reply:
This can be removed.